examples/chat_app/main.py

```diff
@@ -9,11 +11,35 @@
 
 
 async def update(content: ui.column) -> None:
+    # Note: Messages should come from a database
+
+    sent = False
+    # 'sent' should be determined based on the current user
+    # which requires session/user auth - outside of scope
+    # of this example
+    # For now we just alternate for every new message
+
     content.clear()
     with content:  # use the context of each client to update their ui
         for name, text in messages:
-            ui.markdown(f'**{name or "someone"}:** {text}').classes('text-lg m-2')
-        await ui.run_javascript(f'window.scrollTo(0, document.body.scrollHeight)', respond=False)
+            # A simple way to show a message:
+            # ui.markdown(f'**{name or "someone"}:** {text}').classes('text-lg m-2')
+
+            # A more advanced example is using quasar chat_message:
+            sent = not sent
+            if sent:
+                ui.chat_message(text=text,
+                                name=name,
+                                sent=sent,
+                                stamp=datetime.utcnow().isoformat()).classes('w-full')
+            else:
+                ui.chat_message(text=text,
+                                name=name,
+                                sent=sent,
+                                avatar="https://cdn.quasar.dev/img/avatar3.jpg",
```

Comment:
Maybe avatar images can be generated with https://getavataaars.com/ or similar.

Reply:
That would be a solution. But this brings in online/external dependencies. How about leaving this up to the developer? I think that this could be an improvement added later on.

Reply:
I did mean to use a local static resource for the avatar, but let's go with robohash for this example.